enforcer-rules/src/site/apt/requireJavaVendor.apt.vm

```diff
@@ -53,7 +56,13 @@ Require Java Vendor
             <configuration>
               <rules>
                 <requireJavaVendor>
-                  <name>AdoptOpenJDK</name>
+                  <excludes>
+                    <exclude>Pivotal</exclude>
```

Comment:
are these the actual vendor names we would see?

Reply:
These are the names I took from jdk.dev

My current mvn -v returns the following:

```
Java version: 15.0.1, vendor: N/A, runtime: /usr/local/Cellar/openjdk/15.0.1/libexec/openjdk.jdk/Contents/Home
```